code/modules/reagents/chemistry/reagents/pyrotechnic.dm

```diff
@@ -43,6 +45,7 @@
 	reagent_state = SOLID
 	color = "#673910" // rgb: 103, 57, 16
 	process_flags = ORGANIC | SYNTHETIC
+	taste_message = "metal"
```

Comment:
I've zero personal experience, but I believe you wouldn't be able to taste the rust.